ci-operator/step-registry/acm/policies/openshift-plus/acm-policies-openshift-plus-commands.sh

```diff
@@ -21,7 +21,9 @@ sleep 120
 # wait for policies to be compliant
 RETRIES=40
 for try in $(seq "${RETRIES}"); do
-  if [[ $(oc get policies -n policies) != *"NonCompliant"* ]]; then
+  results=$(oc get policies -n policies)
```

Comment:
(optional) This could alternatively be more targeted to only output the `compliant` field from the status:
```suggestion
  results=$(oc get policies -n policies -o jsonpath='{range .items[*].status.status[*]}{.compliant}{"\n"}{end}')
```

Reply:
Oh, I missed that `set -x` was configured. Works for me!